### PR TITLE
Add support for getter and setter

### PR DESCRIPTION
**Pull request by AnEmortalKid**
Adds `git` -> `get` and `sit` -> `set` .

Also fixed `grows` being in the wrong directory, which caused the tests to ignore them. Caught this through deflea since it loads everything under `test` and not just `test/spec`.

### CODE REVIEW

File: LANGUAGE.md [hunk 1]
```diff
@@ -275,6 +275,36 @@ classy Rectangle grows Shape
...
+```
+
+##### Setter
+
+The `sit` keyword binds an object property to a function to be called when there is an attempt to set that property.
+
+```dogescript
+classy Rectangle
+  shh Setter
+  set length with length
```
Comment (AnEmortalKid) [inline LANGUAGE.md]: Should be sit

**Comment by AnEmortalKid** [comment]
> I think this should be split into separate branches for the different changes. git and sit are fine together, but this also includes some other changes (including locking a dependency version?)

will do.

I think the dependency locking was done for an issue with node 10, let me see if I can get the other PR's travis build to also fail.

File: lib/parser.js [hunk 2]
```diff
@@ -109,7 +110,9 @@ var valid = [
     'amaze',
     'classy',
     'maker',
-    'grows'
+    'grows',
+    'git',
+    'sit'
```
Comment (vpzomtrrfrt) [inline lib/parser.js]: do we want to keep a trailing comma here?

File: lib/parser.js [hunk 3]
```diff
@@ -1450,6 +1453,14 @@ function parseStatement(parseContext) {
     return handleMaker(parseContext);
   }
 
+  if (tokens[0] === 'git') {
+    return classHandlers.handleGit(parseContext);
+  }
+
+  if (tokens[0] === 'sit' ) {
```
Comment (vpzomtrrfrt) [inline lib/parser.js]: whitespace?

Comment (AnEmortalKid) [reply lib/parser.js]: Ah, got it, the others are `if(x === 'foo')`. I should probably get a linter on this....

File: lib/parser.js (hunk 2 above)
Comment (AnEmortalKid) [reply lib/parser.js]: Yeah, I agree. I was in the mindset of fixing everything and didn't consider to split all my changes.

**Comment by AnEmortalKid** [comment]
@vpzomtrrfrt anything else you'd like me to tackle here?

**Pull request merged**